### ISSUE TITLE
Port doesn't get bound 

### ISSUE THREAD

**Issue by creepinson** [onyxcode/py-auth-api#5]
Currently, if you enter 
`PORT=6969 python3 start.py` it complains with this error:
```
    sock.bind((host, port))
TypeError: an integer is required (got type str)
```

This is probably because you have to cast the os.getenv to an int via something like `int(PORT)`

**Comment by creepinson** [comment]
Also, I ran it without the port manually specified at the beginning and it says its bound on 8000

**Comment by creepinson** [comment]
Might make prs for these issues but idk about right now

**Comment by creepinson** [comment]
Closed by #6
